build/docker/Release.dockerfile

```diff
@@ -48,6 +62,5 @@ RUN set -eux && \
     apk del gnupg openssl && \
     rm -rf /root/.gnupg
 
-USER vault
-
-ENTRYPOINT ["/bin/vault-k8s"]
+USER ${NAME}
+ENTRYPOINT /bin/${NAME}
```

Comment:
I'm getting an error when running this image:

```
[~/Git/vault-helm/test/acceptance] kubectl logs vault-agent-injector-5bf495b9d-j6kcw -n acceptance
Usage: vault-k8s [--version] [--help] <command> [<args>]

Available commands are:
    agent-inject    Vault Agent injector service
    version         Prints the version
```

I think it might be required to have the `[]` around the command.  I'm assuming that `NAME` is being set to `vault-k8s`?

Reply:
Thanks, yeah I have removed the ${NAME} variable altogether as it doesn't make sense now that this dockerfile lives in this repo.